scripts/flow/environment.js

```diff
@@ -281,3 +281,9 @@ declare module 'node:worker_threads' {
     port2: MessagePort;
   }
 }
+
+declare var Bun: {
```

Comment:
We should really have separate libdefs for each environment but since they're already all mixed together in this one file already I just tacked it on here. Can improve later.